### PR TITLE
Post 0.6 fixes

### PR DESCRIPTION
**Pull request by j00bar**
* Create source with Satellite hostname, not UUID
* Make UUID handling idempotent
* Create application connected to Source

### CODE REVIEW

**Comment by ares** [comment]
@adamruzicka thoughts?

File: tasks/extract_orgs.yml [hunk 1]
```diff
@@ -44,11 +49,9 @@
       mode: 0400
       src: key.pem.j2
   - name: Generate a UUID for this Receptor node
-    command: "uuidgen"
-    register: uuidgen_output
-  - name: Set Receptor Node ID fact
-    set_fact:
-      receptor_uuid: "{{ uuidgen_output.stdout }}"
+    shell: "uuidgen > {{ receptor_config_dir }}/rh_{{ sat_account_id }}/uuid"
```
Comment (adamruzicka) [inline tasks/extract_orgs.yml]: Is uuidgen always installed?

File: tasks/register_with_sources.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@
     client_key: "{{ receptor_config_dir }}/{{ account_dir }}/key.pem"
     method: "POST"
     body:
-      name: "Satellite {{ satellite_uuid }}"
+      name: "{{ satellite_url|urlsplit('hostname') }}"
```
Comment (adamruzicka) [inline tasks/register_with_sources.yml]: Do names of things in sources need to be unique? What if I had two different satellites in two different subnets, but for ease of management they would have the same hostname resolvable only in that network?

Wouldn't it be safer in general to use a variable with fallback to hostname?

File: tasks/register_with_sources.yml [hunk 3]
```diff
@@ -58,10 +58,42 @@
...
+    method: "POST"
+    body:
+      source_id: "{{ source_id }}"
+      application_type_id: "{{ application_type_id }}"
+    body_format: json
+    status_code:
+      - 201
+  when: existing_application_response.json.meta.count == 0
+
+
```
Comment (adamruzicka) [inline tasks/register_with_sources.yml]: nitpick: Trailing empty lines

File: tasks/extract_orgs.yml (hunk 1 above)
Comment (j00bar) [reply tasks/extract_orgs.yml]: It's part of util-linux. It's safe to say it's installed on a RHEL box.

**Pull request merged**